### ISSUE TITLE
nxGroup: does not remove group members when specifying that group should be empty

### ISSUE THREAD

**Issue by eehret** [Azure/nxtools#53]
#### Details of the scenario you tried and the problem that is occurring
We need to create a group and ensure that it remains empty.  The group creation succeeds and it is empty initially , but if users get added to the group nxGroup continues to report the resource as compliant when it should not be compliant, and it does not remove the users from the group.

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
    nxGroup sugroup_exists_and_is_empty {
      Ensure =  'Present'
      GroupName =  'sugroup'
      Members = @()
    }
```

#### The operating system the target node is running
Ubuntu 20.04 LTS

#### Version and build of PowerShell the target node is running
7.3.10

#### Version of the DSC module that was used
1.4.0

**Comment by gurrajatwal** [comment]
Fix merged and will be released in January. Please use the following configuration for this effect.

```
nxGroup sugroup_exists_and_is_empty {
    Ensure =  'Present'
    GroupName =  'sugroup'
    Members = @()
    Force = $true
}
```

**Comment by gurrajatwal** [comment]
The fix has been released as part of nxtools 1.5.0.
https://www.powershellgallery.com/packages/nxtools/1.5.0
